Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionContextCocoa.mm

```diff
@@ -497,6 +497,9 @@ static _WKWebExtensionContextError toAPI(WebExtensionContext::Error error)
     if (permissions.isEmpty())
         return;
 
+    MatchPatternSet matchPatterns;
+    firePermissionsEventListenerIfNecessary(notificationName, permissions, matchPatterns);
```

Comment:
Is there a better way to do this? Like pass in an empty set in the function call? If not, I can split this function into two, one for match patterns and the other for permissions if it seems weird to create these sets but not really use them.